.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,40 +1,27 @@
-<!--- Hi, and thanks for contributing! -->
-<!--- Make sure to provide a general summary of your changes in the title above. -->
-<!--- For example: "[scsynth] Fix crash when encountering cute kittens" -->
+<!-- Please see CONTRIBUTING.md for guidelines. -->
 
 Purpose and Motivation
 ----------------------
 
-<!--- Why is this change required? What problem does it solve? -->
-<!--- If it fixes an open issue, please link to it here by writing "Fixes #555". -->
+Fixes #
 
-Types of changes
-----------------
+Type of change
+--------------
 
-<!--- What types of changes does your pull request introduce? -->
-<!--- Some examples are below (you can delete the lines that don't apply): -->
+<!-- Delete lines that don't apply -->
 
-- Documentation (non-code change which corrects or adds documentation for existing features)
-- Bug fix (non-breaking change which fixes an issue)
-- New feature (non-breaking change which adds functionality)
-- Breaking change (fix or feature that would cause existing functionality to change)
+- Documentation
+- Bug fix
+- New feature
+- Breaking change
 
-Checklist
----------
+To-do list
+----------
 
-<!--- Complete an item by checking it: [x] -->
+<!-- Complete an item by checking it: [x] -->
+<!-- Delete lines that don't apply, add new entries to track your progress -->
 
-- [ ] All previous tests are passing
-- [ ] Tests were updated or created to address changes in this PR, and tests are passing
-- [ ] Updated documentation, if necessary
+- [ ] Code is tested
+- [ ] All tests are passing
+- [ ] Updated documentation
 - [ ] This PR is ready for review
-
-<!--- See DEVELOPING.md for instructions on running and writing tests. -->
-
-Remaining Work
---------------
-
-<!--- If any work remains to be done, please give a brief description here. -->
-<!--- Consider providing a todo-list so we can easily track completion progress. -->
-
-<!--- Thanks for contributing! -->
```

Comment:
:+1: except maybe leave this line in?

Reply:
i'm not wild about it personally... if we want to be nice to new contributors, let's do that by leaving nice comments rather than automated template messages!